.analysis_options

```diff
@@ -21,6 +21,7 @@ analyzer:
   language:
```

Comment:
@tvolkert - @pq is out of town so I'm going to assume your LGTM in this file covers this. :-)